candle-examples/examples/quantized/main.rs

```diff
@@ -106,65 +93,21 @@ impl Which {
             | Self::Mistral7b
             | Self::Mistral7bInstruct
             | Self::Mistral7bInstructV02 => true,
+            _ => false,
```

Comment:
The long explicit pattern matching was on purpose, this forces someone adding a new case to decide whether this function should return false or true rather than defaulting to one of the two cases. Same for the other functions.

Reply:
I will revert.